IntegrationTests/UI.IntegrationTests/CommandsDialogs/FormBrowseTests.cs

```diff
@@ -321,9 +313,9 @@ private void RunFormTest(Func<FormBrowse, Task> testDriverAsync)
                 testDriverAsync);
         }
 
-        private static void WaitForRevisionsToBeLoaded(FormBrowse form)
+        private static void WaitForRevisionsToBeLoaded(FormBrowse form, [CallerMemberName] string caller = "")
         {
-            UITest.ProcessUntil("Loading Revisions", () => form.GetTestAccessor().RevisionGrid.GetTestAccessor().IsDataLoadComplete);
+            UITest.ProcessUntil($"{caller} loading revisions", () => form.GetTestAccessor().RevisionGrid.GetTestAccessor().IsDataLoadComplete, maxMilliseconds: 10_000);
```

Comment:
Is 10s reasonable or abort earlier?

Reply:
This is just the maximum waiting time. The revisions shall and are loaded more quickly. Though 1.5 seconds did not suffice too often.

Reply:
If 1.5 seconds may not be enough, should we bump it to say 3 seconds instead of 10? Are we being overzealous?

Reply:
These testcases _take_ much time on AppVeyor. 5 seconds might be enough. But I do not want to waste another day just to notice that 1.5 seconds are overzealous.
Again: The execution time is not affected if it finishes in 1.5 or 3 seconds regardless of whether the timeout is 3, 5 or 60 seconds! But with 10 seconds we tolerate occasional hiccups of the AppVeyor VMs.